frontend/views/containers/proposals/ProposalTemplate.vue

```diff
@@ -82,11 +82,12 @@
     template(slot='footer' v-if='!isConfirmation')
       .c-footer
         i.icon-vote-yea
-        span(v-if='groupShouldPropose' v-safe-html='footerVotingExplanation')
+        span(v-if='shouldPropose' v-safe-html='footerVotingExplanation')
         i18n(
-          v-else
+          v-else-if='!groupShouldPropose'
           :args='LTags("strong")'
         ) Your group has less than 3 members, so {strong_}this change will be immediate{_strong} (no voting required).
+        slot(v-else name='skipToProposeFooter')
```

Comment:
Like I mentioned in `DistributionDate.vue`, slot should be removed and then these block can be reverted back to `v-if` and `v-else` pair.